sw/device/silicon_creator/lib/drivers/rstmgr.c

```diff
@@ -30,10 +32,39 @@ static void rstmgr_alert_info_collect(void) {
 }
 
 uint32_t rstmgr_reason_get(void) {
+  // Static assertions for bitfield indices.
+#define REASON_ASSERT(index, expect) \
+  static_assert((index) == (expect), #index " value incorrect.");
+  REASON_ASSERT(kRstmgrReasonPowerOn, RSTMGR_RESET_INFO_POR_BIT);
```

Comment:
nit: can you add a line after the definition?

Reply:
Done